website/docs/advanced/builder.md

```diff
@@ -0,0 +1,211 @@
+
+## 3. Is Builder Configured?
+
+When a validator is proposing a block, the following is checked before attempting to use the Builder through the relay.
+- `--http-mev-relay` flag was provided and is pointed to mev-boost or an active relay of the correct network
+- circuit breaker is not triggered 
+- validator is registered (beacon API was successfully called to save to local storage)
+
+If all checks are satisfied then 5. Builder via Relay URL will be used to get the execution payload which contains the transactions and build a blinded block. However, if the checks do not pass then the beacon node will fall back to 4. Local Execution Client.
+
+
+## 4. Local Execution Client: kept in sync and up to date
+
+local execution clients such as `geth` or `nethermind` must continue to run as usual even while using a builder and will be used in case the builder does not pass the `Is Builder Configured?` check. The execution client should be synced and running alongside your beacon node and earnings from the block will be compared to the earnings from the builder's payload. If the local execution payload fails then the entire function will fail.
+
+## 5. Builder: connected via relay URL
+
+The ETHStaker community provides a list of some of the relays that can be used as well as any censorship they may have [here](https://github.com/eth-educators/ethstaker-guides/blob/main/MEV-relay-list.md). You can also run your own locally such as MEV boost but each relay on the list will have their own instructions on how to run. If running your own instead of using a provided url due to latency, you will simply need to update your `--http-mev-relay` flag on your beacon node with the appropriate url for the specific network in use. The relay will connect to a builder which connects to block searchers. 
+
+:::info
+
+Make sure you are using the correct version that supports the current version of the beacon node. Hard-forks will typically require updates to relays.
+
+
+:::
+
+</TabItem>
+<TabItem value="remove">
+
+## 1. Validator Client: unregister validator
+
+Update the following configurations and restart the validator client to stop periodic registration of the validator. 
+- remove the `--enable-builder` flag.
+- remove all wanted references of the `builder` field from the associated file/json for the validators you no longer want to register with in the `--proposer-settings-file` and `--proposer-settings-url` flag.
+
+### validator registration expiration
+
+:::caution
+
+There is currently an issue with un-registering validators in the db, it's suggested to re-register while using the `--enable-registration-cache` flag on the beacon node so that registrations can properly unregister.
```

Comment:
```suggestion
There is currently an issue with unregistering validators in the db, it's suggested to re-register while using the `--enable-registration-cache` flag on the beacon node so that registrations can properly unregister.
```